ignite/services/network/networkchain/networkchain.go

```diff
@@ -287,7 +287,7 @@ func (c Chain) NodeID(ctx context.Context) (string, error) {
 func (c Chain) CheckConfigVersion() error {
 	configPath := c.chain.ConfigPath()
 	if configPath == "" {
-		return config.ErrConfigNotFound
+		return nil
```

Comment:
This change is unclear.  I think we should keep returning the error and check it in the calling function.